meetings/2018-06-20-EuropeanXFEL/ProgressReports/Sheffield-Leeds.md

```diff
@@ -37,10 +37,15 @@ Science, Bioinformatics, and Materials Science.
 - The previously generated CoCalc tutorial was extended by adding tutorial sections for students having courses in CoCalc as well as with a hands-on tutorial for lecturers to get started. The material can be found as a website at [https://tutorial.cocalc.com/](https://tutorial.cocalc.com/) (The repository for this is located at:[https://github.com/sagemathinc/cocalc_tutorial](https://github.com/sagemathinc/cocalc_tutorial)) ([D2.17](https://github.com/OpenDreamKit/OpenDreamKit/issues/252),
 [T2.6](https://github.com/OpenDreamKit/OpenDreamKit/issues/29)).
 - Developed a [Jekyll template](https://github.com/trallard/Modules-template)
-for use by academics and researchers using Jupyter notebooks for course materials and dissemination.  This will allow them to easily produce course websites based on Jupyter notebooks connected to cloud computing resources such as Microsoft Azure Notebooks ([D2.17](https://github.com/OpenDreamKit/OpenDreamKit/issues/252),
+for use by academics and researchers using Jupyter notebooks for course materials and dissemination. Such a template allows the creation of Jupyter notebooks based websites using Jekyll, which is the
```

Comment:
That's very nice; would you have some time left to add a new "Use case: authoring course material" to our [use case](https://opendreamkit.org/project/use-cases/) collection, which would include a link to the template in the section "ODK contribution"?

Reply:
Just a random thought: should the template have binder links when the notebooks are not connected to Azure?

Reply:
In general, please have a look at the use cases, and suggest (or add directly when possible!) items to "how ODK contributed".

Reply:
Sure I will add this as a test case. I had originally planned to add a regular post to the site but having this as a use case makes much more sense. 

Re. binder: this could be a nice workaround. If there is no Azure library added then a binder launch button can be auto-generated. 

I will file this as an issue on the template repository and sort it out